tests/unit/calendar/calendar_core.js

```diff
@@ -139,7 +139,7 @@ asyncTest( "keyboard handling", function() {
 			.focusGrid( element )
 			.simulate( "keydown", { keyCode: $.ui.keyCode.LEFT } );
 		setTimeout(function() {
-			$( ":focus" ).simulate( "keydown", { keyCode: $.ui.keyCode.ENTER } );
+			$( document.activeElement ).simulate( "keydown", { keyCode: $.ui.keyCode.ENTER } );
```

Comment:
The `focusGrid` call above returns the to-be-focused element. Could store that in a variable and reuse it here. Applies to many other places as well. Not sure if its better to avoid a temp var at the cost of accessing globals.